libafl/src/stages/calibrate.rs

```diff
@@ -27,9 +29,10 @@ where
     I: Input,
     O: MapObserver,
     OT: ObserversTuple<I, S>,
-    S: HasCorpus<I> + HasMetadata,
+    S: HasCorpus<I> + HasMetadata + HasNamedMetadata,
 {
     map_observer_name: String,
+    map_name: String,
```

Comment:
I think here, we need map_observer_name only (?)
we want the map observer, and the feedbackstate with the same name.

Reply:
I would argue there can be multiple feedbacks built on top of a single observer

Reply:
However, it may be better in general to hand in a ref to the Observer, here 🤔

Reply:
after all this time, you still have to get how the borrow checker works lol

Reply:
No, why? Just pass in the observer to read the name from it, then drop the borrow again

Reply:
why sould you pass feedback and observer when feedback already contains this info?

Reply:
or you should pass observer + feedback name as string

Reply:
To create feedback, as we already do.
Nevermind didn't mean to confuse anyone 😅 it's good as it is